platform/emscripten/CMakeLists.txt

```diff
@@ -0,0 +1,54 @@
+cmake_minimum_required(VERSION 3.8)
+
+set(CMAKE_GENERATOR "Ninja")
+
+if(NOT DEFINED ENV{EMSDK})
+    message(FATAL_ERROR "Environment variable EMSDK is not defined. Follow the README steps and source emsdk_env.* from the EMSDK root directory in this shell before running CMake")
+endif()
+
+set(CMAKE_GENERATOR "Ninja")
+set(CMAKE_TOOLCHAIN_FILE ../../vendor/vcpkg/scripts/buildsystems/vcpkg.cmake)
+set(VCPKG_TARGET_TRIPLET "wasm32-emscripten")
+
+set(VCPKG_CHAINLOAD_TOOLCHAIN_FILE $ENV{EMSDK}/upstream/emscripten/cmake/Modules/Platform/Emscripten.cmake)
+
+project(mbgl-wasm LANGUAGES C CXX)
+
+set(CMAKE_CXX_STANDARD 17)
+
+find_package(libuv CONFIG REQUIRED)
```

Comment:
Remove - it seems like it builds it fine using wasm32-emscripten as target then the linker fails miserably